x/txreward/internal/types/Funder.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type Funder struct {
+	Address sdk.AccAddress `json:"address"`
+}
+
+type Funders []Funder
```

Comment:
This type can be replaced by using `ctypes.Addresses` where `ctypes` is the `types` of the `common` module:

```golang
	ctypes "github.com/commercionetwork/commercionetwork/x/common/types"
```

I've just moved this type from the `docs` module into the new `common` for simplicity